### PR TITLE
#2371 fix to display utc date at Datetime picker

### PR DESCRIPTION
**Pull request by yamatoya**
#2371
* datetime picker display changed from local time to utc.
* reference : [bootstrap-datetimepicker::options::timeZone](https://github.com/Eonasdan/bootstrap-datetimepicker/blob/master/docs/Options.md#timezone)

### CODE REVIEW

**Comment by grobie** [comment]
Please run `make assets` and add the changed bindata file here.

**Comment by grobie** [comment]
Awesome! Works great. I'd suggest to mention the timezone in the hover box here: https://github.com/prometheus/prometheus/blob/8ebeed0b44280705d4eb4a2431e8546fd2978b76/web/ui/static/js/graph_template.handlebar#L80

For example `End time of graph (UTC)`

File: web/ui/static/js/graph.js [hunk 1]
```diff
@@ -120,6 +120,7 @@ Prometheus.Graph.prototype.initialize = function() {
     showTodayButton: true,
     showClear: true,
     showClose: true,
+    timeZone: 'UTC'
```
Comment (grobie) [inline web/ui/static/js/graph.js]: Please keep the trailing comma.

**Comment by beorn7** [comment]
Can we merge this into release-1.7 instead of master first?
This is a fairly important fix.

**Comment by beorn7** [comment]
@yamatoya With the right Github incantation, it should be easily possible to rebase this on top of the release-1.7 branch. Could you do that, please? We can then integrate it into the imminent 1.7.2 release, and I will also forward-merge into master so that 1.8 has it, too.

**Comment by yamatoya** [comment]
@grobie
add the changed bindata file
change title
add comma

@beorn7 
changed merge into release-1.7

**Comment by beorn7** [comment]
Thanks @yamatoya but you pulled in all the commits from master, too.

You have to rebase your own commits on top of the release-1.7 branch and then force-push them to make this PR work.

**Comment by yamatoya** [comment]
I took mistake git operation and automatically PR closed.
I recreate PR #3182

**Pull request closed without merging**